exercises/isbn-verifier/canonical-data.json

```diff
@@ -149,6 +149,15 @@
             },
             "expected": false
         },
+        {
+            "uuid": "6ec400c2-6421-11ec-97c5-8c1645feb539",
+            "description": "extraneous invalid characters are not ignored",
```

Comment:
To better distinguish this case from the preceding case (  
ed6e8d1b-382c-4081-8326-8b772c581fec​), it may be good to follow the same pattern used in other cases for the description:

E.g.: `"description": "invalid characters are not ignored in valid isbn"`

Reply:
Makes sense. Updated the description as suggested.